### PR TITLE
disabled option added

### PR DESCRIPTION
**Pull request by muratcorlu**
`active` option added to easily disable adding headers on production environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following table describes the properties of the `options` object.
 |---------------|-----------------------------------------|--------|-------------|
 | `name`        | name of the server timing metric        | String | 'mw'        |
 | `description` | description of the server timing metric | String | `undefined` |
+| `active`      | set `false` if you want to disable      | Boolean| `true`      |
```
Comment (cvazac) [inline README.md]: Thanks for the submission!!

Would you be willing to change this to a `disabled` flag instead? I personally think it makes it more obvious that it is truly optional. WDYT?

Comment (muratcorlu) [reply README.md]: Makes sense. I'll change it.

**Comment by muratcorlu** [comment]
I renamed `active` option as `disabled` as requested.

**Review by cvazac — APPROVED** [review]
Thanks for the contribution! FYI before I ship this, I'm going to change a bit of the README.

**Comment by cvazac** [comment]
published as 3.1.0

**Pull request merged**